jekyll.yml

```diff
@@ -108,6 +108,11 @@ defaults:
     values:
       layout: 'docs-v2'
 
+  - scope:
+      path: 'kong-ingress-controller'
```

Comment:
We have some naming inconsistency: "kubernetes ingress controller" vs "kong ingress controller" vs "kong for kubernetes". Is it within the scope of this PR to assume "kong ingress controller" as our choice for all communication?